src/status_im2/common/bottom_sheet/view.cljs

```diff
@@ -49,7 +49,10 @@
            (hide translate-y bg-opacity window-height))))))
 
 (defn view
-  [{:keys [hide? insets]} {:keys [content override-theme selected-item]}]
+  [{:keys [hide? insets]}
+   {:keys [content override-theme selected-item enable-scroll? padding-bottom-override]
+    :or   {enable-scroll?          true
+           padding-bottom-override nil}}]
```

Comment:
If `padding-bottom-override` is not passed in the map, its value will be `nil`, hence setting a default of `nil` seems unnecessary.